docs/font-inspector.html

```diff
@@ -261,9 +261,19 @@ <h1>Free Software</h1>
 }
 
 function onFontLoaded(font) {
+    if (window.font) {
+        window.font.onGlyphUpdated = null;
+    }
     window.font = font;
     renderText(font);
     displayFontData(font);
+    font.onGlyphUpdated = (glyphId) => {
```

Comment:
That's an interesting idea using events to trigger redrawing. I would pefer to make this more versatile so we can use it in other places as well (colr/cpal updates, glyph outline changes, etc.) Something like this: https://medium.com/@aryakvn/emitting-events-from-custom-classes-in-javascript-ea635d8023c6

I would normally tend to do that in a separate PR, however, if we provide `onGlyphUpdated` for now, we'd at least have to document it, just to deprecate it when implementing the optimized event handling... On the other hand, it will still be a while until 2.0.0 and we'll probably be able to implement the event handling before release... I'm unsure and would like to get input by @ILOVEPIE @yne

Reply:
Agree,
I'm in no hurry to move to a centralized event handling and do more lazy parsing. So the current onGlyphUpdated API seems fine